packages/records/lib/models/records.ts

```diff
@@ -0,0 +1,417 @@
+import dayjs from 'dayjs';
```

Comment:
`models/records.ts` contains all the functions that are reading/writing from/to the records table